pandas/core/config_init.py

```diff
@@ -957,3 +960,53 @@ def register_converter_cb(key):
         styler_environment,
         validator=is_instance_factory([type(None), str]),
     )
+
+
+merge_chunk_size_doc = """
+: int | tuple[int, int]
+    Conditional merge chunked cross merge with deferred filter, chunk size.
```

Comment:
Nit: IMO clearer as `Chunk size of merge where on is a callable`

Reply:
done